README.md

```diff
@@ -3,41 +3,180 @@
 [![Apache License Version 2.0](https://img.shields.io/badge/license-Apache_2.0-green.svg)](LICENSE)
 ![GitHub branch checks state](https://img.shields.io/github/checks-status/openvinotoolkit/openvino/master?label=GitHub%20checks)
 ![Azure DevOps builds (branch)](https://img.shields.io/azure-devops/build/openvinoci/b2bab62f-ab2f-4871-a538-86ea1be7d20f/13?label=Public%20CI)
+[![PyPI Status](https://badge.fury.io/py/openvino.svg)](https://badge.fury.io/py/openvino)
 [![PyPI Downloads](https://pepy.tech/badge/openvino)](https://pepy.tech/project/openvino)
 
-This toolkit allows developers to deploy pre-trained deep learning models
-through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
+## Contents:
+
+ - [What is OpenVINO?](#what-is-openvino-toolkit)
+    - [Components](#components)
+ - [Supported Hardware matrix](#supported-hardware-matrix)
+ - [License](#license)
+ - [Documentation](#documentation)
+ - [Tutorials](#tutorials)
+ - [Products which use OpenVINO](#products-which-use-openvino)
+ - [System requirements](#system-requirements)
+ - [How to build](#how-to-build)
+ - [How to contribute](#how-to-contribute)
+ - [Get a support](#get-a-support)
+ - [See also](#see-also)
+
+## What is OpenVINO toolkit?
+
+OpenVINO™ toolkit allows developers to deploy pre-trained deep learning models through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
```

Comment:
Could we use an official description from documentation? This definition was align and approved last year with all stakeholders.

OpenVINO™ is an open-source toolkit for optimizing and deploying AI inference.
- Boost deep learning performance in computer vision, automatic speech recognition, natural language processing and other common tasks
- Use models trained with popular frameworks like TensorFlow, PyTorch and more
- Reduce resource demands and efficiently deploy on a range of Intel® platforms from edge to cloud